suites/pacific/cephadm/test_5-2_new-feature_apply-spec.yaml

```diff
@@ -0,0 +1,148 @@
+#===============================================================================================
+#----- Ceph code changes in 5.2 ------
+# Tier-level: 1
+# Test-Suite: test_5-2_new-feature_apply-spec.yaml
+# Test-Case: Deploy all new feature services at 5.2 using a spec file option as below command
+#               eg., ceph orch apply -i <spec_file.yaml>
+#
+# Cluster Configuration:
+#    cephci/conf/pacific/cephadm/tier-1_5node_cephadm_bootstrap.yaml
+#
+# Test Steps :
+#   (1) Bootstrap cluster with options,
+#       - skip-monitoring-stack: true
+#       - orphan-initial-daemons: true
+#       - registry-json: <registry-URL>
+#       - fsid: <cluster-fsid>
+#       - mon-ip: <monitor IP address: Required>
+#   (2) Copy SSH keys to nodes.
+#   (3) Add nodes to cluster with address and role labels attached to it using Host spec yaml file.
+#   (4) Deploy monitoring stack and services using apply spec option, (" ceph orch apply -i <spec_file>")
+#       - 3 Mon on node1, node2, node3 using host placements.
+#       - MGR using placement using label(mgr).
+#       - 3 OSD - addition of OSD's with raw OSD on node2 and node3
+#===============================================================================================
+tests:
+  - test:
+      name: Install ceph pre-requisites
+      desc: installation of ceph pre-requisites
+      module: install_prereq.py
+      abort-on-fail: true
+  - test:
+      name: Cephadm Bootstrap
+      desc: cephadm cluster bootstrap
+      module: test_bootstrap.py
+      config:
+        command: bootstrap
+        base_cmd_args:
+          verbose: true
+        args:
+          skip-monitoring-stack: true
+          orphan-initial-daemons: true
+          registry-json: registry.redhat.io
+          custom_image: true
+          mon-ip: node1
+          fsid: f64f341c-655d-11eb-8778-fa163e914bcc
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
+      name: Host addition with spec file
+      desc: add hosts using spec file.
+      module: test_cephadm.py
+      config:
+        steps:
+        - config:
+            service: host
+            command: set_address
+            args:
+              node: node1
+        - config:
+            service: host
+            command: label_add
+            args:
+              node: node1
+              labels: apply-all-labels
+        - config:
+            command: apply_spec
+            service: orch
+            specs:
+             - service_type: host
+               address: true
+               labels: apply-all-labels
+               nodes:
+                 - node2
+                 - node3
+             - service_type: host
+               address: true
+               labels: apply-all-labels
+               nodes:
+                 - node4
+             - service_type: host
+               address: true
+               labels: apply-all-labels
+               nodes:
+                 - node5
+      abort-on-fail: true
+  - test:
```

Comment:
Let us include basic IOs as part of this test suite post service deployment with spec.